src/viam/sdk/module/service.cpp

```diff
@@ -229,8 +229,9 @@ void ModuleService_::close() {
     }
 }
 
-void ModuleService_::add_api_from_registry(std::shared_ptr<Server> server, API api) {
-    const std::lock_guard<std::mutex> lock(lock_);
+void ModuleService_::_add_api_from_registry_inlock(std::shared_ptr<Server> server,
```

Comment:
I think convention in this repo is trailing `_` for private, so I'd call this `add_api_from_registry_inlock_`